source/_docs/upstream-updates.md

```diff
@@ -165,3 +165,35 @@ There are multiple reasons that 503 errors might occur when updating:
 - PHP segfault: These are tricky to troubleshoot because very little debugging information is present. A temporary fix is available. Contact Pantheon Customer Support if you think you have been affected.
 
 - Timeouts are another cause of 503 errors, though they are much less likely to occur if you are using the Pantheon domains. If the operation takes more than 60 seconds, you might see a timeout occur.
+
+### Process to Resolve Conflicts Locally responds with 'Already-up-to-date'
+If you know your site's core codebase does not match Pantheon's upstream, and attempts to resolve it locally do not work because your site repository is seen as up-to-date, you may need to manually overwrite core files. We only recommend this method as an absolute last resort.
+

```

Comment:
We might want to add some common reasons this happens. The most common that I see is that a customer has pulled updates, but subsequently, they've applied a revert commit. We also might want to add a warning to notify them that all patches to core will be overwritten